matsim/src/main/java/org/matsim/core/population/PersonUtils.java

```diff
@@ -102,13 +102,22 @@ public static String getCarAvail(Person person) {
         return (String) person.getAttributes().getAttribute(CAR_AVAIL);
     }
 
+    /**
+     * Car is not available if Person.getLicense gives "no" or if
```

Comment:
the function actually uses an `AND` operator

Reply:
The comment expresses the negation of the code, which is imo easier to understand as text.